.travis.yml

```diff
@@ -18,6 +19,7 @@ env:
     - MESOS_NATIVE_JAVA_LIBRARY=/usr/lib/libmesos.so
     - CLJ_HTTP_ASYNC_POOL_TEST_DURATION_MULTIPLIER=5
   matrix:
+    - TEST_DIR=executor DEPS_CMD='python3 setup.py install' TEST_CMD='python3 setup.py nosetests'
```

Comment:
The integration test uses `python setup.py nosetests`. I'm being more explicit with `python3`